src/proof/lean/lean_post_processor.cpp

```diff
@@ -545,9 +545,37 @@ bool LeanProofPostprocessCallback::update(Node res,
       break;
     }
     // Arith
-    case PfRule::ARITH_SUM_UB:
     case PfRule::ARITH_MULT_POS:
     case PfRule::ARITH_MULT_NEG:
+    {
+      AlwaysAssert((int) args.size() == 2);
```

Comment:
```suggestion
      Assert(args.size() == 2);
```
Better to use Assert since it's compiled out in production code. And you do not need the cast.